### PR TITLE
Default status / NO DATA docs

### PR DESCRIPTION
**Pull request by Dalje-et**
### What does this PR do?
Documents the behaviour of the on_missing_data in combination with timeout_h and group_retention.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/create/configuration.md [hunk 1]
```diff
@@ -120,18 +120,63 @@ See the documentation for [process check][1], [integration check][2], and [custo
...
+
+| Option                    | Monitor status & notification                                             |
+|---------------------------|---------------------------------------------------------------------------|
+| `Evaluate as zero`        | Empty result is "replaced" with zero and compared to the alert/warning thresholds. For example, if the alert threshold is set to `> 10`, a zero would not trigger that condition, hence the Monitor status is set to `OK`.   |
+| `Show last known status`  | The last known status of the group/monitor is set.                        |
+| `Show NO DATA`            | Monitor status is set to `NO DATA`.                                       |
+| `Show NO DATA and notify` | Monitor status is set to `NO DATA` and a notification is sent out.        |
+| `Show OK`                 | Monitor is resolved and status is set to `OK`.                            |
+
+The options `Evaluate as zero` and `Show last known status` are shown depending on the type of the query:
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:167]:
```suggestion
The `Evaluate as zero` and `Show last known status` options are displayed based on the query type:
```

File: content/en/monitors/create/configuration.md [hunk 2]
```diff
@@ -120,18 +120,63 @@ See the documentation for [process check][1], [integration check][2], and [custo
...
+
+**Evaluate as zero**
+This option is available for monitors using `Count` queries.
+
+**Show last known status**
+On the other hand, `Show last known status` is available for monitors using any other query type than `Count`, for example `Gauge`, `Rate`, or `Distribution`.
+
+Both options are considered as the **default status** of the monitor.
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:175]:
```suggestion
Both options are the default status of the monitor.
```

File: content/en/monitors/create/configuration.md [hunk 3]
```diff
@@ -120,18 +120,63 @@ See the documentation for [process check][1], [integration check][2], and [custo
...
+| `Show NO DATA and notify` | Monitor status is set to `NO DATA` and a notification is sent out.        |
+| `Show OK`                 | Monitor is resolved and status is set to `OK`.                            |
+
+The options `Evaluate as zero` and `Show last known status` are shown depending on the type of the query:
+
+**Evaluate as zero**
+This option is available for monitors using `Count` queries.
+
+**Show last known status**
+On the other hand, `Show last known status` is available for monitors using any other query type than `Count`, for example `Gauge`, `Rate`, or `Distribution`.
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:173]:
```suggestion
- **Evaluate as zero:** This option is available for monitors using `Count` queries.
- **Show last known status:** This option is available for monitors using any other query type than `Count`, for example `Gauge`, `Rate`, and `Distribution`.
```

File: content/en/monitors/create/configuration.md [hunk 4]
```diff
@@ -120,18 +120,63 @@ See the documentation for [process check][1], [integration check][2], and [custo
...
+{{% tab "Other monitor types" %}}
+
+If data is missing for `N` minutes:
+
+- `Evaluate as zero` / `Show last known status`
+- `Show NO DATA`
+- `Show NO DATA and notify`
+- `Show OK`.
+
+{{< img src="/monitors/create/on_missing_data.png" alt="No Data Options" style="width:70%;">}}
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:155]:
```suggestion
{{< img src="/monitors/create/on_missing_data.png" alt="No Data Options" style="width:70%;">}}

- `Evaluate as zero` / `Show last known status`
- `Show NO DATA`
- `Show NO DATA and notify`
- `Show OK`.
```

File: content/en/monitors/create/configuration.md [hunk 5]
```diff
@@ -142,6 +187,20 @@ For some metrics that report periodically, it may make sense for triggered alert
 
 In most cases this setting is not useful because you only want an alert to resolve once it is actually fixed. So, in general, it makes sense to leave this as `[Never]` so alerts only resolve when the metric is above or below the set threshold.
 
+#### Group retention time
+
+Drop the group from the monitor status after `N` hours of missing data (can be set up to 72 hours).
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:192]:
```suggestion
You can drop the group from the monitor status after `N` hours of missing data. The maximum length of time is 72 hours).
```

File: content/en/monitors/create/configuration.md [hunk 6]
```diff
@@ -142,6 +187,20 @@ For some metrics that report periodically, it may make sense for triggered alert
...
+
+{{< img src="/monitors/create/group_retention_time.png" alt="Group Retention Time Option" style="width:70%;">}}
+
+Similar to the [Auto-resolve option][4], the group retention works when data is no longer being submitted. This option controls how long the group is kept in the monitor's status once data stops reporting. By default, a group is kept for 24h in the status before it's dropped. The start time of the group retention as well as the Auto-resolve option are **identical**, namely as soon as the monitor query returns no data.
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:196]:
```suggestion
Similar to the [Auto-resolve option][4], the group retention works when data is no longer being submitted. This option controls how long the group is kept in the monitor's status once data stops reporting. By default, a group keeps the status for 24 hours before it is dropped. The start time of the group retention and the Auto-resolve option are **identical** as soon as the monitor query returns no data.
```

File: content/en/monitors/create/configuration.md [hunk 7]
```diff
@@ -142,6 +187,20 @@ For some metrics that report periodically, it may make sense for triggered alert
...
+
+Typical use cases for defining a group retention time are:
+- you would like to drop the group immediately/shortly after data stops reporting
+- you would like to keep the group in the status for as long as you usually take for troubleshooting
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:200]:
```suggestion
Some use cases to define a group retention time include:

- When you would like to drop the group immediately or shortly after data stops reporting
- When you would like to keep the group in the status for as long as you usually take for troubleshooting
```

File: content/en/monitors/create/configuration.md [hunk 8]
```diff
@@ -164,3 +223,5 @@ The time (in seconds) to delay evaluation. This should be a non-negative integer
 [1]: /monitors/create/#monitor-types
 [2]: /monitors/notify/variables/?tab=is_alert#tag-variables
 [3]: /monitors/notify/#renotify
+[4]: /monitors/create/configuration#auto-resolve
+[5]: /monitors/create/configuration/?tab=othermonitortypes
```
Comment (alai97) [inline content/en/monitors/create/configuration.md:227]:
```suggestion
[5]: /monitors/create/configuration/?tabs=thresholdalert#no-data
```

**Review by alai97 — CHANGES_REQUESTED** [review]

File: content/en/monitors/create/configuration.md (hunk 2 above)
Comment (Dalje-et) [reply content/en/monitors/create/configuration.md:175]: Removing this sentence alltogether.

**Review by alai97 — APPROVED** [review]

**Pull request merged**